Additional (non-standard) claims in WIT and WPT (#151)
ietf-wg-wimse/draft-ietf-wimse-s2s-protocol: draft-ietf-wimse-s2s-protocol.md

thus, `Workload-Identity-Token`, `workload-identity-token`, `WORKLOAD-IDENTITY-TOKEN`,
etc., are all valid and equivalent header field names. However, case is significant in the header field value.

### Including Additional Claims {#add-claims}

> The WIT is a JSON structure and therefore can be trivially extended by adding more claims than are defined in the current specification.

Reviewer: the WIT is not a JSON structure. it contains some JSON but is not JSON

Reviewer: github says this is outdated but who is it to day what's in style or not?

This, however, could result in interoperability issues, which the following rules are addressing.

* Deployers may freely add claims to the WIT beyond those defined here, in closed environments. Such claims SHOULD be collision-resistant, such as `example.com/myclaim`.
* A recipient that does not understand such claims MUST ignore them, as per Sec. 4 of {{RFC7519}}.

> * Outside of closed environments, new claims MUST be registered with IANA {{IANA.JWT.CLAIMS}} before they can be used.

Reviewer: seems like these rules are not aligned with similar content from RFC 7519 itself, which doesn't seem good

Author: @bc-pi In what way?

thus, `Workload-Identity-Token`, `workload-identity-token`, `WORKLOAD-IDENTITY-TOKEN`,
etc., are all valid and equivalent header field names. However, case is significant in the header field value.

### Including Additional Claims {#add-claims}

> The WIT is a JSON structure and therefore can be trivially extended by adding more claims beyond those defined in the current specification.

Reviewer: ```suggestion
The WIT contains JSON structures and therefore can be trivially extended by adding more claims beyond those defined in the current specification.
```
